### PR TITLE
Vimar french pr

### PR DESCRIPTION
**Pull request by vimar**
## Motivation/Description of the PR
- Add French translation to CodeceptJS

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [x] translation
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [x] :rocket: New functionality
- [ ] :bug: Bug fix
- [x] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [] Tests have been added
- [x] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Awesome, thank you! :fr:

File: docs/translation.md [hunk 1]
```diff
@@ -66,6 +66,26 @@ Scenario('Efetuar login', (Eu) => {
...
+To write your tests in French you can enable the French translation by adding to config:
+
+```json
+  "translation": "fr-FR"
+```
+
+Now you can write tests like this:
+
+```js
+Fonctionnalité('Se connecter sur GitHub', (io) => {
```
Comment (DavertMik) [inline docs/translation.md:80]: Oups, I just figured out that this example is not correct.
According to the translation file, it should be:

```js
Scénario('Se connecter sur GitHub', (io) => {
```

@vimar could you quick fix this?

Comment (DavertMik) [reply docs/translation.md:80]: Also, why it is `io` if you use `Ja` ?

File: docs/translation.md [hunk 2]
```diff
@@ -66,6 +66,26 @@ Scenario('Efetuar login', (Eu) => {
...
+    Je.suisSurLaPage('https://github.com/login');
+    Je.remplisLeChamp("Username or email address", "jean-dupond");
+    Je.remplisLeChamp("Password", "*********");
+    Je.cliqueSur("Sign in");
+    Je.Vois("Learn Git and GitHub without any code!");
```
Comment (DavertMik) [inline docs/translation.md:85]: also it should be `vois` (lowercased)

File: docs/translation.md (hunk 1 above)
Comment (vimar) [reply docs/translation.md:80]: A copy/paste error :disappointed: 
I fix this in the new pr: https://github.com/codeceptjs/CodeceptJS/pull/2578

**Pull request merged**